wcivf/apps/peoplecvs/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+from people.models import Person
+
+
+class CV(models.Model):
+    """
+    A candidate's CV.
+    """
+    person = models.ForeignKey(Person)
```

Comment:
I think this can be a OneToOneField, unless there are likely to be more than one CV per person? Does the CVs API support this?